packages/fiori-freestyle-writer/templates/listdetail/add/webapp/controller/List.controller.ts

```diff
@@ -125,7 +125,10 @@ export default class List extends BaseController {
      * and group settings and refreshes the list binding.
      */
     public onRefresh() {
-        this.list.getBinding("items")?.refresh(false);
+        const items = this.list.getBinding("items");
+        if (items){
```

Comment:
```suggestion
        if (items) {
```
Make it consistent with the rest?

Reply:
Done in https://github.com/SAP/open-ux-tools/pull/928/commits/1e8135ab0a32fc14a34815b4dfa1ab877afe4eb3